src/main/java/sungshin/sooon/domain/post/Post.java

```diff
@@ -0,0 +1,36 @@
+package sungshin.sooon.domain.post;
+
+import lombok.*;
+import org.hibernate.annotations.DynamicUpdate;
+import sungshin.sooon.domain.account.Account;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@EqualsAndHashCode(of = "post_id")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@DynamicUpdate
+public class Post {
+
+    @Id @GeneratedValue
```

Comment:
https://github.com/SSWU-Community/spring-study/pull/7#discussion_r675366943
```suggestion
    @Id @GeneratedValue(strategy= GenerationType.IDENTITY)
```